Tests/SonarQube.TeamBuild.PreProcessor.Tests/RoslynAnalyzerProviderTests.cs

```diff
@@ -334,8 +384,8 @@ private static WellKnownProfile CreateValidCSharpProfile()
 
   <Deployment>
     <NuGetPackages>
-      <Package Id=""SonarLint"" Version=""1.3.0"" />
-      <Package Id=""Wintellect.Analyzers"" Version=""1.0.5.0-rc1"" />
+      <NuGetPackage Id=""SonarLint"" Version=""1.3.0"" />
+      <NuGetPackage Id=""Wintellect.Analyzers"" Version=""1.0.5.0-rc1"" />
```

Comment:
Correcting the element names to match the spec (didn't matter before because the packages weren't being checked).